Bottom: 67149233ae53f270b6636e590bd57d753fb89e35
Top:    025cbd27a9d76891bc232640fb51ee268da3e017
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-09 17:30:17 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index d7b5e34..ae0b9fa 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
